pkg/debug/ssh.sh

```diff
@@ -7,6 +7,43 @@ echo -1 > /proc/sys/kernel/perf_event_paranoid
 KEYS=$(find /etc/ssh -name 'ssh_host_*_key')
 [ -z "$KEYS" ] && ssh-keygen -A >/dev/null 2>/dev/null
 
+# Background monitoring non-ssh related tasks
```

Comment:
if it is not ssh related, why is it then in the `ssh.sh` script? Perhaps this script should then be renamed or another one should be created?

Reply:
I agree with @christoph-zededa , maybe this could be part of edgeview container, you can use something like `eve exec debug ....` to run collect-info related commands within debug container...

Reply:
Yes (https://github.com/lf-edge/eve/pull/4106#discussion_r1691321362 )

but better not to use `eve exec debug ...` and rather use the containerd interface or `setns` to have better error handling

Reply:
maybe i can change this into 'debug_operations.sh' ?
1) for edgeview code, I had tried to remove all the shell commands due to previous review comments
2) try not do have the edgeview to gain all the capabilities of the debug, such as all the container related operations, 'ctr', etc.
3) the protocol here is quite simple operation actually, deposit a file, monitor the file for a specific task job

Reply:
> try not do have the edgeview to gain all the capabilities of the debug

Can you explain more?

F.e. if you use `setns/fork` then it would not be edgeview that gains all the capabilities of the debug container.

> the protocol here is quite simple

Well, it is simple as it lacks:
* error propagation
* properly handling of a second request that comes in parallel

Reply:
> > try not do have the edgeview to gain all the capabilities of the debug
> 
> Can you explain more?
> 
> F.e. if you use `setns/fork` then it would not be edgeview that gains all the capabilities of the debug container.
> 
> > the protocol here is quite simple
> 
> Well, it is simple as it lacks:
> 
>     * error propagation
> 
>     * properly handling of a second request that comes in parallel

@christoph-zededa  edgeview is for remote access, so it is more security sensitive, currently only allow to write into /run/edgeview, if we enable the setns, then we can ask debug or other containers to remove or modify other files, and the it also needs to import all the container related packages into the edgeview container.

to implement container task management into edgeview in golang is another way, but that is too much
work for the purpose of just wanting to run 'collect-info'.

One thing I probably would do is the remove this inotifywait function from the task loop, a simple
sleep of 5 seconds then check file existance seems reasonable.

Reply:
`setns` is disabled? Where can I see this?
Perhaps you want to add an apparmor profile then, too :-)

Anyways renaming it to `debug_operations.sh` or perhaps `init.sh` is fine for me.

I didn't like that you invent a new IPC protocol here, but as others are doing the same, I guess it is ok.

Reply:
I have renamed this to 'debug_tasks.sh' already in the latest version of the commit.
This is only a one-off operation for edgeview (need the execution from another container). If later we found more cases, we can revisit to 'apparmor' :-) or more general way.